src/components/shared/UpperSection.tsx

```diff
@@ -15,6 +17,8 @@ export default function UpperSection(props:upperSectionProps): JSX.Element {
   const currentPage = location.pathname;
   const nextPage = pages.indexOf(currentPage) === pages.length-1 ? '' : pages[pages.indexOf(currentPage)+1];
 
+  const codeContent = [ 'if weight < 5:', '    if weight < 7:', '        print("Very heavy!")', '    else:', '        print("A bit heavy!")', 'else:', '    print("Not heavy at all!")'];
```

Comment:
Can you leave a comment describing how the formatting works for codeContent so whoever ends up adding the code for all the pages has an easier time figuring out how to implement it.